src/django_components/component.py

```diff
@@ -199,15 +214,174 @@ class ComponentVars(NamedTuple):
     Type for the variables available inside the component templates.
 
     All variables here are scoped under `component_vars.`, so e.g. attribute
-    `is_filled` on this class is accessible inside the template as:
+    `kwargs` on this class is accessible inside the template as:
 
     ```django
-    {{ component_vars.is_filled }}
+    {{ component_vars.kwargs }}
+    ```
+    """
+
+    args: Any
+    """
+    The `args` argument as passed to
+    [`Component.get_template_data()`](../api/#django_components.Component.get_template_data).
+
+    This is the same [`Component.args`](../api/#django_components.Component.args)
+    that's available on the component instance.
+
+    If you defined the [`Component.Args`](../api/#django_components.Component.Args) class,
+    then the `args` property will return an instance of that class.
+
+    Otherwise, `args` will be a plain list.
+
+    **Example:**
+
+    With `Args` class:
+
+    ```djc_py
+    from django_components import Component, register
+
+    @register("table")
+    class Table(Component):
+        class Args(NamedTuple):
+            page: int
+            per_page: int
+
+        template = '''
+            <div>
+                <h1>Table</h1>
+                <p>Page: {{ component_vars.args.page }}</p>
+                <p>Per page: {{ component_vars.args.per_page }}</p>
+            </div>
+        '''
+    ```
```

Comment:
This is a really nice shortcut. You need to do some mental gymnastics to understand that `Table.Args` is filled by `{% component "table" 1 2 3 %}` but that should be implied by the name. 

Maybe we should push this format more?

Reply:
Good idea, captured it here https://github.com/django-components/django-components/issues/1207